### PR TITLE
Mp test fixes

### PR DESCRIPTION
**Pull request by AI0867**

### CODE REVIEW

File: utils/travis/mp_test_executor.sh [hunk 1]
```diff
@@ -12,10 +15,49 @@ hostpid=$!
...
-wait $hostpid
+START_TIME=$SECONDS
+HOST_RUNNING=0
+JOIN_RUNNING=0
+while true; do
+    # Timeout
+    EXEC_TIME=$(($SECONDS - $START_TIME))
+    if [ $EXEC_TIME -gt $TIMEOUT_TIME ]; then
+        kill $hostpid 2>/dev/null
+        kill $joinpid 2>/dev/null
```
Comment (jyrkive) [inline utils/travis/mp_test_executor.sh]: For consistency, I'd prefer if you added a `break` statement here as well.

File: utils/travis/mp_test_executor.sh [hunk 2]
```diff
@@ -12,10 +15,49 @@ hostpid=$!
...
+    fi
+    # Check if clients still running
+    if ! kill -0 $hostpid 2>/dev/null; then
+        HOST_RUNNING=1
```
[the remark this answers is not in the document]
Comment (AI0867) [reply utils/travis/mp_test_executor.sh]: I'm using the shell convention of 0 = true and anything else = false.

File: utils/travis/mp_test_executor.sh (hunk 1 above)
Comment (AI0867) [reply utils/travis/mp_test_executor.sh]: Looks like I missed that one. Added.

File: utils/travis/mp_test_executor.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (AI0867) [reply utils/travis/mp_test_executor.sh]: Annoyingly, github is showing the wrong comment as referring to an outdated diff.

[the remark this answers is not in the document]
Comment (jyrkive) [reply utils/travis/mp_test_executor.sh]: OK, fine. I approved the PR.

[the remark this answers is not in the document]
Comment (AI0867) [reply utils/travis/mp_test_executor.sh]: I decided to change it to strings, for clarity.

**Pull request merged**